receiver/jmxreceiver/config.go

```diff
@@ -112,5 +115,25 @@ func (c *Config) validate() error {
 	if c.OTLPExporterConfig.Timeout < 0 {
 		return fmt.Errorf("%v `otlp.timeout` must be positive: %vms", c.ID(), c.OTLPExporterConfig.Timeout.Milliseconds())
 	}
+
+	if parsed, err := c.parseProperties(); err != nil {
+		return fmt.Errorf("%v %w", c.ID(), err)
```

Comment:
```suggestion
		return fmt.Errorf("failed parsing properties: %w", err)
```
I think when this gets logged it includes the receiver already as a field so not sure it needs to be in the error message. But if it does I might do something like
```go
fmt.Errorf("failed parsing properties for receiver %v: %w", c.ID(), err)
```
Usually the error's are separated by `:`.

Reply:
You're correct that the receiver id is logged with the failure, but after making this a map of strings validation is no longer necessary.